services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/store/device/TableAdapterStore.java

```diff
@@ -81,23 +81,26 @@ public TableAdapterStore(final JDBCClient client, final Tracer tracer, final Sta
 
 
     /**
-     * Read a device using {@link #readDevice(io.vertx.ext.sql.SQLOperations, DeviceKey, Span)} and the
+     * Read a device using {@link #readDevice(io.vertx.sqlclient.SqlConnection, DeviceKey, Span)} and the
      * current SQL client.
      *
      * @param key The key of the device to read.
      * @param span The span to contribute to.
      *
-     * @return The result from {@link #readDevice(io.vertx.ext.sql.SQLOperations, DeviceKey, Span)}.
+     * @return The result from {@link #readDevice(io.vertx.sqlclient.SqlConnection, DeviceKey, Span)}.
      */
     protected Future<ResultSet> readDevice(final DeviceKey key, final Span span) {
```

Comment:
I think we can delete this method as it seems to not be used anymore

Reply:
Done.